lib/config/validation.spec.ts

```diff
@@ -913,4 +913,123 @@ describe('config/validation', () => {
       ]);
     });
   });
+
+  describe('globalOptions()', () => {
+    it('validates options of string type', async () => {
```

Comment:
@rarkins you're a native speaker, which is the correct format?

1. validates options of string type
2. validates options of type string
3. validates string type options
4. validates options with type string

I suspect `3` is the correct format, but I'm asking you first to make sure. 😄

Reply:
All are ok for me for a test description

Reply:
i like 3